prog/mutation.go

```diff
@@ -169,7 +169,7 @@ func (ctx *mutator) mutateArg() bool {
 		if len(ma.args) == 0 {
 			return false
 		}
-		arg, ctx := ma.pseudoRandomArgs(r)
+		arg, ctx := ma.chooseArgument(r)
```

Comment:
This is change is not related to this commit, it's better to merge it into the previous one (that is about arg mutation priorities).